Jenkinsfile

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Contiki Applications
+ *
+ */
+def examples = [
+    'hello-world', 
+    'ping-ipv6',
+    'trickle-library',
+    'servreg-hack',
+    'http-socket',
+    'udp-ipv6',
+    'er-rest-example',
+    'webserver-ipv6',
+]
+
+def mikroe_apps = [
+    'sample-app',
+    'test-button',
+    'test-i2c',
+    'test-led',
+    'test-spi',
```

Comment:
I believe test-spi was added just to test the failure case, it will be removed in the final commit